board.py

```diff
@@ -2,7 +2,7 @@
 from os import system
 import random
 
-compro_d = 0
+binary_count = 0
```

Comment:
What's the usage of this variable?

Reply:
I use that as a conditional, in case the player wants to play another round, this variable will be equal to 1 and the cards can be cleared from the table. And when all this happens, it will be equal to 0.